renderdoc/driver/ihv/nv/nv_counters.cpp

```diff
@@ -0,0 +1,380 @@
+#include "nv_counters.h"
```

Comment:
The two new files need the standard MIT License copyright comment at the top, with just `Copyright (c) 2018 Baldur Karlsson`